### PR TITLE
Create section for desktop wallet

### LINKED ISSUE #51: Desktop wallet set-up guide
Create a guide for how to get it and how to start using it

### PR DESCRIPTION
**Pull request by candida-d**
Closes #51

### CODE REVIEW

File: docs/tools/vega-wallet/desktop-app/latest/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+Get the latest version of the Vega Wallet desktop app from the [GitHub releases page](https://github.com/vegaprotocol/vegawallet-desktop/releases). 
+
+Keep track of where you've saved the file, because you will need to know where to look for it to use it.
+
+Click through the tabs below to find out which download file you need. 
+
+<Tabs groupId="operating-systems">
+<TabItem value="windows" label="Windows">
+
+Download `vegawallet-windows-amd64.zip`
```
Comment (ValentinTrinque) [inline docs/tools/vega-wallet/desktop-app/latest/getting-started.md:24]:
```suggestion
Download `vegawallet-desktop-windows-amd64.zip`
```

File: docs/tools/vega-wallet/desktop-app/latest/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+</TabItem>
+<TabItem value="mac" label="MacOS">
+
+Download `vegawallet-darwin-amd64.zip`
```
Comment (ValentinTrinque) [inline docs/tools/vega-wallet/desktop-app/latest/getting-started.md:29]:
```suggestion
Download `vegawallet-desktop-darwin-amd64.zip`
```

File: docs/tools/vega-wallet/desktop-app/latest/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+For Macs with an M1 processor (released since November 2020): 
+
+Download `vegawallet-darwin-arm64.zip`
```
Comment (ValentinTrinque) [inline docs/tools/vega-wallet/desktop-app/latest/getting-started.md:33]:
```suggestion
Download `vegawallet-desktop-darwin-arm64.zip`
```

File: docs/tools/vega-wallet/desktop-app/latest/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  
+</TabItem>
+
+<TabItem value="linux" label="Linux">
+
+Download `vegawallet-linux-amd64.zip`
```
Comment (ValentinTrinque) [inline docs/tools/vega-wallet/desktop-app/latest/getting-started.md:39]:
```suggestion
Download `vegawallet-desktop-linux-amd64.zip`
```

File: docs/tools/vega-wallet/desktop-app/latest/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+2. Click on "Use recovery phrase"
+3. Enter your recovery phrase, exactly in the order that it was first presented to you. 
+4. Choose your wallet version (1 or 2). 
+5. Choose a Vega network to connect to: Click on the arrow in the bottom bar of the app. 
+
+## Troubleshooting
+The Vega Wallet desktop app does not yet have the full functionality of the CLI wallet app.
+
+### Not supported
+To do any of the following, you will need to use the **[CLI wallet](docs/tools/vega-wallet/cli-wallet/)**:
```
Comment (ValentinTrinque) [inline docs/tools/vega-wallet/desktop-app/latest/getting-started.md:71]: For consistency:

```suggestion
To do any of the following, you will need to use the **[wallet CLI](docs/tools/vega-wallet/cli-wallet/)**:
```

Sometimes we have "CLI wallet" et "wallet CLI".

**Review by ValentinTrinque — CHANGES_REQUESTED** [review]

File: docs/tools/vega-wallet/desktop-app/latest/getting-started.md (hunk 5 above)
Comment (candida-d) [reply docs/tools/vega-wallet/desktop-app/latest/getting-started.md:71]: I try to use CLI wallet for the app (and generally in the docs) - is there somewhere you saw "wallet CLI", as I'd probably change that one instead

**Pull request merged**